canonical/ubuntu.com
# WD-5646 - Removing toggles + Landscape Trial

## Done
Fixes [WD-5646](https://warthogs.atlassian.net/browse/WD-5646)

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- visit [/pro/dashboard](https://ubuntu-com-13081.demos.haus/pro/dashboard)
- make sure the dashboard matches the new designs


[WD-5646]: https://warthogs.atlassian.net/browse/WD-5646?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
